src/temp/Explore/CustomSlider.tsx

```diff
@@ -22,7 +18,7 @@ export const SmallSlider = styled(Slider)(() => ({
   },
   '.MuiSlider-track': {
     height: '2.5px',
-    color: `${exploreRangeSliderColorNew}`,
+    color: `rgba(235, 87, 87)`,
```

Comment:
Instead of hard coding use this format : `${'rgba(var(--explore-range-background-new))'}` here.